mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -124,7 +124,7 @@ def write(self, payload, log=False):
             self.selected_resource.write(payload)
             if log:
                 self.logger.prn_txd(payload)
-        return payload
+        return True
```

Comment:
Ok no problem. Please rebase with master. Sorry I just merged the other PR #130. You may have to fix merge issues.